G/GDAL/build_tarballs.jl

```diff
@@ -66,6 +66,7 @@ rm -f ${prefix}/lib/*.la
     --with-expat=$prefix \
     --with-zstd=$prefix \
     --with-sqlite3=$prefix \
+    --with-pg=$prefix \
```

Comment:
The builds are failing with `configure: error: Only --with-pg=yes/no supported`.
Could you try that?